Products/CMFQuickInstallerTool/QuickInstallerTool.py

```diff
@@ -90,6 +118,7 @@ class QuickInstallerTool(UniqueObject, ObjectManager, SimpleItem):
 
     meta_type = 'CMF QuickInstaller Tool'
     id = 'portal_quickinstaller'
+    # toolicon = 'skins/plone_images/product_icon.png'
```

Comment:
@mauritsvanrees leftover? 🙂

Reply:
Removed now. I have copied the PNG icon from CMFPlone to `tool.png`.